<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="curr_box"/>
        </item>
        <item>
         <widget class="QComboBox" name="year"/>
        </item>
        <item>
         <widget class="QComboBox" name="month"/>
        </item>
        <item>
         <widget class="QPushButton" name="search_btn_2">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
